docs/book/src/developer/architecture/controllers/metadata-propagation.md

```diff
@@ -3,23 +3,49 @@ Cluster API controllers implement consistent metadata (labels & annotations) pro
 This behaviour tries to be consistent with Kubernetes apps/v1 Deployment and ReplicaSet.
 New providers should behave accordingly fitting within the following pattern: 
 
+![](../../../images/metadata-propagation.jpg)
```

Comment:
Q: Can the jpg be modified? If yes, can we add an html comment here to hint at how (e.g. with which tool)